test/stream.test.js

```diff
@@ -446,11 +446,11 @@ test('return a 404 if the stream emits a 404 error', t => {
 })
 
 test('should support send module 200 and 404', t => {
-  t.plan(8)
+  t.plan(7)
   const fastify = Fastify()
 
   fastify.get('/', function (req, reply) {
-    const stream = send(req.raw, __filename)
+    const stream = Readable.from(['Text stream'])
```

Comment:
Removed `send` usage here as this was inferring a `content-type` from the stream it was creating which hid the nature of this change.

Reply:
since this test is specifically about the send module, we should not change it I believe, and keep the existing assertion. iow, restore this whole test to what it used to be